### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by Berkmann18** [all-contributors/all-contributors-cli#137]
- `all-contributors-cli` version: 5.7.0
- `node` version: 11.0.0
- `npm` (or `yarn`) version: 6.5.0

Relevant code or config

N/A

What you did:
Generated the contributors list

What happened:

It generated the list as expected.

Reproduction repository:

`all-contributors generate`

Problem description:

When the contributors table is generated, the images (read `<img .../>` tags) are missing an `alt` attribute which gets flagged on accessibility linters.
The reason for that is that images with no `alt` attribute can be hard/impossible to read on devices/systems for those with accessibility issues where page contents are read to them (think if the image wasn't visible, what would you rather see: nothing or a text explaining the image?).

Suggested solution:
It's not that big of an issue however as a developer who cares about accessibility, I think it would be a good idea to ensure that `img` tags include something like: `alt="ContributorName"` or something.

I'll open a PR in a few hours.

**Comment by Berkmann18** [comment]
It seems that there are quite a few failing tests that aren't related to this PR so I'll try fixing them on another branch and then rebasing the one for the PR to make sure the build succeeds.
